Stripe/STPAddressFieldTableViewCell.m

```diff
@@ -262,7 +262,7 @@ - (void)nextTapped:(__unused id)sender {
 #pragma mark - STPFormTextFieldDelegate
 
 - (void)formTextFieldTextDidChange:(STPFormTextField *)textField {
-    if (self.type != STPAddressFieldTypeCountry) {
+    if (self.type != STPAddressFieldTypeCountry && self.type != STPAddressFieldTypeSEPACountry) {
```

Comment:
The logic here prevents setting `self.contents` to the text field's text (which will be the country's display name, e.g. "Germany"). In `delegateCountryCodeDidChange:`, we set the cell's contents to the country code.

The bug was that we were doing this for cells of type `STPAddressFieldTypeCountry` (which we use in the card form), but not `STPAddressFieldTypeSEPACountry` (which we use in the SEPA form), so in the SEPA form, `cell.contents` was `Germany` rather than `DE`.